src/main/resources/jenkins/plugins/office365connector/WebhookJobProperty/help-macros.html

```diff
@@ -0,0 +1,9 @@
+<div align="help">
+  This section defines additional conditions that must be met to send the notification.
+  It consists from pair of template and value. If template written
+  as <a href="https://wiki.jenkins.io/display/JENKINS/Token+Macro+Plugin">token macro</a>
```

Comment:
... but I believe this link and getting familiar with the plugin makes this feature understandable.